### DISCUSSION TITLE
No result found for `cmake: CMAKE_INSTALL_LIBDIR`

### DISCUSSION THREAD

**Discussion by zchrissirhcz** [zealdocs/zeal#1634] [Q&A] [answered]
I would like to get search result by typing `cmake: CMAKE_INSTALL_LIBDIR` in zeal but no result found:

<img width="336" alt="截屏2024-06-23 15 35 00" src="https://github.com/zealdocs/zeal/assets/3831847/f18b42e4-05c7-40c4-aaa5-80c2726a130b">

I can find result in the online, official cmake document:

https://cmake.org/cmake/help/latest/search.html?q=CMAKE_INSTALL_LIBDIR
<img width="1065" alt="截屏2024-06-23 15 33 54" src="https://github.com/zealdocs/zeal/assets/3831847/fec8aeb2-6601-4dda-aec2-bfcbc8d81032">

Zeal version: `v0.7.1-dev.19+db781ed`

CMake docset: the latest

As shown above, the result is matched in text, instead of a "keyword/command/variable" of cmake. I'm not sure if this fulltext search result can be supported by docset.

@Kapeli @trollixx

**Comment by trollixx** [comment, marked as answer]
Zeal doesn't have full text search, and `CMAKE_INSTALL_LIBDIR` has no dedicated paragraph in the docs and hence no symbol in the index. There's nothing wrong with the docset in this case.
